### PR TITLE
Adding structured answer documentation

### PR DESCRIPTION
**Pull request by mintsoft**
https://github.com/duckduckgo/duckduckgo-documentation/issues/175

cc @moollaza @jagtalon

### CODE REVIEW

File: duckduckhack/goodie/goodie_advanced_handle_functions.md [hunk 1]
```diff
@@ -15,6 +15,26 @@ handle remainder => sub {
 }
 ```
 
+## Using Structured Responses
+
+Most Goodies essentially take some input and provide output based from it, for example the [Calculator Goodie](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/Calculator.pm).
```
Comment (jagtalon) [inline duckduckhack/goodie/goodie_advanced_handle_functions.md]: What do you think about "Most Goodies essentially take some input and provide output based from it. One example is the ..."? I think there should be a period after "it."

File: duckduckhack/goodie/goodie_advanced_handle_functions.md [hunk 2]
```diff
@@ -15,6 +15,26 @@ handle remainder => sub {
...
+
+As this is often used, Goodies can returned a structured answer which will always be displayed in a consistent way between goodies. A simple example is as follows:
```
Comment (jagtalon) [inline duckduckhack/goodie/goodie_advanced_handle_functions.md]: "can returned" => should be "can return"

**Comment by jagtalon** [comment]
Love it. Thanks @mintsoft

**Comment by mintsoft** [comment]
Sorted @jagtalon

**Comment by jagtalon** [comment]
@mintsoft Nice :+1:

**Pull request merged**